index.html

```diff
@@ -291,30 +316,46 @@ <h3>File Names</h3>
             <h3>MiniApp Resources</h3>
             <section id="sec-miniapp-resources-html">
                 <h3>HTML Resources</h3>
-                <p>MiniApp user interfaces, including the visual components that defines the structure and the interaction elements of the pages, are described in HTML resources.</p>
-                <p>HTML resources MUST meet all the following criteria:</p>
+                <p><dfn id="dfn-html-resource" data-lt="MiniApp HTML resource|MiniApp HTML resources|HTML resource" data-dfn-type="dfn">HTML resources</dfn> are documents that contain markup and information to define MiniApp user interfaces, including the structure, visual components, and user interaction elements.</p>
+                <p>[=HTML resources=] MUST meet all the following criteria:</p>
                 <dl class="conformance-list">
+                    <dt>Syntax and content</dt>
+                    <dd>An [=HTML resource=] MUST use the <a href="https://html.spec.whatwg.org/multipage/xhtml.html#the-xhtml-syntax">XML syntax</a> defined in [[HTML]], and the profile of MiniApp Content Documents.</dd>
```

Comment:
What's the rationale of using the [XML syntax](https://html.spec.whatwg.org/multipage/xhtml.html#the-xhtml-syntax) (instead of the [HTML syntax](https://html.spec.whatwg.org/multipage/syntax.html#syntax))?

Reply:
I think we still need to clarify the final format and syntax (not fully agreed by the group yet). It's really an early proposal based on the previous discussions. I understand there are some MiniApp vendors whose markup needs strict XML in the document. No strong opinion on this and happy to modify it based on the group's feedback. Empathizing with developers, I would prefer HTML syntax instead of XHTML :)

Reply:
While we discussing, to loosen constraints, I've reverted it to HTML syntax.

Reply:
Changes in db0e16b8fdefd394f9e598d56ea1b2f0dcee50e6